compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -759,7 +762,7 @@ CompiledJarFile generate(BIRPackage module, boolean isEntry) {
         // generate object/record value classes
         JvmValueGen valueGen = new JvmValueGen(module, this, methodGen, typeHashVisitor, types);
         JvmCastGen jvmCastGen = new JvmCastGen(symbolTable, jvmTypeGen, types);
-        valueGen.generateValueClasses(jarEntries, jvmConstantsGen, jvmTypeGen);
+        valueGen.generateValueClasses(jarEntries, jvmConstantsGen, jvmTypeGen, isRemoteMgtEnabled);
```

Comment:
Shall we move this to the `JvmValueGen` constructor and set it to a field in `JvmObjectGen` through its constructor? So we don't have to pass this as a parameter inside ValueGen method calls.